tests/api/test_cargo_event_entity.py

```diff
@@ -1,33 +1,26 @@
 from unittest import TestCase
 
-import json
-
 from vortexasdk.api.cargo_movement import CargoPortUnloadEvent
-from vortexasdk.api.geography import GeographyEntity
 
 
 class TestCargoEventEntity(TestCase):
-    def test_serialize(self):
-        with open("tests/api/examples/cargo_event_entity1.json", "r") as f:
-            serialized = json.load(f)
-            deserialized = CargoPortUnloadEvent.parse_obj(serialized)
-
-            expected = CargoPortUnloadEvent(
-                event_type="cargo_port_unload_event",
-                location=[
-                    GeographyEntity(
-                        id="2aaad41b89dfad19e5668918018ae02695d7710bcbe5f2dc689234e8da492de3",
-                        layer="country",
-                        label="United Kingdom",
-                        source="model",
-                        probability=1,
-                    )
+    def test_serialize(self) -> None:
+        CargoPortUnloadEvent.model_validate(
```

Comment:
same here https://github.com/VorTECHsa/python-sdk/pull/493/files#r1575914768

Reply:
I think the test is easier to understand and work on when the actual test data is defined in the same file. I don't think this change has any material effect other than readability?